projects/packages/publicize/src/class-connections-post-field.php

```diff
@@ -266,7 +266,7 @@ public function rest_insert( $post, $request, $is_new ) {
 	protected function get_meta_to_update( $requested_connections, $post_id = 0 ) {
 		global $publicize;
 
-		if ( ! $publicize ) {
+		if ( ! $publicize || ( defined( 'DOING_AUTOSAVE' ) && DOING_AUTOSAVE ) ) {
```

Comment:
I'm wondering if we should also check if the post is published here, so we can continue to save the meta while it's a draft.